client/src/modules/dashboard/Chapters/graphql/mutations.ts

```diff
@@ -27,3 +27,22 @@ export const updateChapter = gql`
     }
   }
 `;
+
+export const changeChapterUserRole = gql`
+  mutation changeChapterUserRole(
+    $chapterId: Int!
+    $roleId: Int!
+    $userId: Int!
+  ) {
+    changeChapterUserRole(
+      chapterId: $chapterId
+      roleId: $roleId
+      userId: $userId
+    ) {
+      chapter_role {
+        id
+        name
+      }
```

Comment:
```suggestion
      id
```
I know we have to return something, but since the returned values aren't used by the client, is `id` alone sufficient?